routers/user.py

```diff
@@ -0,0 +1,259 @@
+async def get_approved_services(user: User = Depends(get_current_user)):
+    user_id = user.access_token.sub
+    token = get_management_token()
+    user_data = await fetch_user_data(user_id, token)
+    services = user_data.get("app_metadata", {}).get("services", [])
+    approved_services = [
+        service for service in services
+        if service.get("status") == "approved"
+    ]
+    return {"approved_services": approved_services}
+
+
+@router.get("/me/services/pending")
+async def get_pending_services(user: User = Depends(get_current_user)):
+    user_id = user.access_token.sub
+    token = get_management_token()
+    user_data = await fetch_user_data(user_id, token)
+    services = user_data.get("app_metadata", {}).get("services", [])
+    pending_services = [
+        service for service in services
+        if service.get("status") == "pending"
+    ]
+    return {"pending_services": pending_services}
+
+
+@router.get("/me/resources")
+async def get_all_resources(user: User = Depends(get_current_user)):
+    user_id = user.access_token.sub
+    token = get_management_token()
+    user_data = await fetch_user_data(user_id, token)
+    services = user_data.get("app_metadata", {}).get("services", [])
+    resources = [
+        resource
+        for service in services
+        for resource in service.get("resources", [])
+    ]
+    return {"resources": resources}
+
+
+@router.get("/me/resources/approved")
+async def get_approved_resources(user: User = Depends(get_current_user)):
+    user_id = user.access_token.sub
+    token = get_management_token()
+    user_data = await fetch_user_data(user_id, token)
+    services = user_data.get("app_metadata", {}).get("services", [])
+    approved_resources = [
+        resource
+        for service in services
+        for resource in service.get("resources", [])
+        if resource.get("status") == "approved"
+    ]
+    return {"approved_resources": approved_resources}
+
+
+@router.get("/me/resources/pending")
+async def get_pending_resources(user: User = Depends(get_current_user)):
+    user_id = user.access_token.sub
+    token = get_management_token()
+    user_data = await fetch_user_data(user_id, token)
+    services = user_data.get("app_metadata", {}).get("services", [])
+    pending_resources = [
+        resource
+        for service in services
+        for resource in service.get("resources", [])
+        if resource.get("status") == "pending"
+    ]
+    return {"pending_resources": pending_resources}
+
+
+@router.get("/me/all/pending")
+async def get_all_pending(user: User = Depends(get_current_user)):
+    user_id = user.access_token.sub
+    token = get_management_token()
+    user_data = await fetch_user_data(user_id, token)
+    services = user_data.get("app_metadata", {}).get("services", [])
+    pending_services = [
+        service for service in services
+        if service.get("status") == "pending"
+    ]
+    pending_resources = [
```

Comment:
I think these should be methods on the `UserData` model, that way we won't have to repeat this code. e.g. have `UserData.get_pending_services()`, `UserData.get_pending_resources()`